charts/kong/templates/_helpers.tpl

```diff
@@ -346,6 +346,10 @@ The name of the service used for the ingress controller's validation webhook
 {{- if .Values.ingressController.admissionWebhook.enabled }}
   {{- $_ := set $autoEnv "CONTROLLER_ADMISSION_WEBHOOK_LISTEN" (printf "0.0.0.0:%d" (int64 .Values.ingressController.admissionWebhook.port)) -}}
 {{- end }}
+{{- if (not (eq (len .Values.ingressController.watchNamespaces) 0)) }}
+  {{- $_ := set $autoEnv "CONTROLLER_WATCH_NAMESPACE" (.Values.ingressController.watchNamespaces | join ",") -}}
+  {{- $_ := set $autoEnv "CONTROLLER_CONTROLLER_KONGCLUSTERPLUGIN" "disabled" -}}
```

Comment:
I'm _kinda_ lost as to how this one is relevant to the watchNamespaces,  did I miss something?

Reply:
When you watch specific namespaces only, you cannot watch any cluster scope resources. If you enable the KCP controller, it attempts to watch cluster-scoped KCPs unsuccessfully.

Reply:
ooop ofc, ty.